website/content/en/preview/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -145,19 +157,63 @@ The new stack has only one user, `admin`, and the password is stored in a secret
 
-If you delete a node with kubectl, Karpenter will gracefully cordon, drain,
-and shutdown the corresponding instance. Under the hood, Karpenter adds a
-finalizer to the node object, which blocks deletion until all pods are
-drained and the instance is terminated. Keep in mind, this only works for
-nodes provisioned by Karpenter.
+### Private Clusters
 
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step15-delete-node.sh" language="bash"%}}
+You can optionally install Karpenter on a [private cluster](https://docs.aws.amazon.com/eks/latest/userguide/private-clusters.html#private-cluster-requirements) using the `eksctl` installation by setting `privateCluster.enabled` to true in your [ClusterConfig](https://eksctl.io/usage/eks-private-cluster/#eks-fully-private-cluster)
 
-### Delete the cluster
-To avoid additional charges, remove the demo infrastructure from your AWS account.
+```bash
+privateCluster:
+  enabled: true
+```
+
+Private clusters have no outbound access to the internet. This means that in order for Karpenter to reach out to the services that it needs to access, you need to enable specific VPC private endpoints. Below shows the endpoints that you need to enable to successfully run Karpenter in a private cluster:
+
+```text
+com.amazonaws.<region>.ec2
+com.amazonaws.<region>.ecr.api
+com.amazonaws.<region>.ecr.dkr
+com.amazonaws.<region>.s3 – For pulling container images
+com.amazonaws.<region>.sts – For IAM roles for service accounts
+com.amazonaws.<region>.ssm - For resolving default AMIs
+com.amazonaws.<region>.sqs - If using the [interruption handling]({{< ref "../../concepts/disruption#interruption" >}}), to access interruption messages
+```
+
+If you do not currently have these endpoints surfaced in your VPC, you can add the endpoints by running
+
+```bash
+aws ec2 create-vpc-endpoint --vpc-id ${VPC_ID} --service-name ${SERVICE_NAME} --vpc-endpoint-type Interface --subnet-ids ${SUBNET_IDS} --security-group-ids ${SECURITY_GROUP_IDS}
+```
+
+{{% alert title="Note" color="primary" %}}
+
+Karpenter (controller and webhook deployment) container images must be in or copied to Amazon ECR private or to a another private registry accessible from inside the VPC. If these are not available from within the VPC, or from networks peered with the VPC, you will get Image pull errors when Kubernetes tries to pull these images from ECR public.
+
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+
+There is currently no VPC private endpoint for the [Price List Query API](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/using-price-list-query-api.html). As a result, pricing data can go stale over time. By default, Karpenter ships a static price list that is updated when each binary is released.
```

Comment:
Need to set isolatedVPC: true in the helm chart too  https://github.com/aws/karpenter/blob/main/charts/karpenter/values.yaml#L190

Reply:
Ah, good callout 👍